apps/web/src/constants/editorEnums.ts

```diff
@@ -0,0 +1,11 @@
+export enum ActivePageEnum {
```

Comment:
moved to the shared folder as this const is used in different modules

Reply:
For the filename we reached an agreement for the hooks, but for general files which pattern we should go?

Reply:
I feel like we should have a `shared` folder where we can keep the constants, enums, and whatever else... wdyt?